backend/tournesol/views/video.py

```diff
@@ -69,6 +74,14 @@ class VideoViewSet(mixins.CreateModelMixin,
     queryset = Video.objects.all()
     pagination_class = LimitOffsetPagination
     permission_classes = [IsAuthenticatedOrReadOnly]
+
+    throttle_classes = [
+        PostScopeRateThrottle,
+        BurstUserRateThrottle,
+        SustainedUserRateThrottle
+    ]
```

Comment:
So here so rate limit is applied for anonymous users on `GET /video/`. Is it deliberate?

Reply:
Fixed.